test/unit-test/core_mqtt_state_utest.c

```diff
@@ -52,8 +52,53 @@ int suiteTearDown( int numFailures )
     return numFailures;
 }
 
+#define  MQTT_STATE_ARRAY_MAX_COUNT    10
```

Comment:
Should this be defined with the other macros above?

Reply:
Done.